timApp/item/routes.py

```diff
@@ -1452,6 +1453,45 @@ def get_item(item_id: int):
     return json_response(i)
 
 
+@view_page.get("/items/accesses/<int:item_id>")
+def get_item_accesses(item_id: int):
+    """Return a list of usergroup ids that have any access to the item"""
+
+    i = Item.find_by_id(item_id)
+    if not i:
+        raise NotExist("Item not found")
+    verify_view_access(i)
+
+    accs = list(set(v.usergroup.id for v in i.block.accesses.values()))
+
+    # from timApp.util.logger import log_info
+    # log_info(str(accs))
+    return json_response(accs)
```

Comment:
This is a bit sketchy to read, otherwise good! More verbose variable names would be useful.

Reply:
For the default setting (user settings toggle), I would say badges could be default on? At least me, I always find myself wondering who is going to see this and this..